### PR TITLE
Deployment preparation: `V3 Stable Pool Factory`

### PR DESCRIPTION
**Pull request by jubeira**
# Deployment preparation: `V3 Stable Pool Factory`

Build-info: https://github.com/balancer/balancer-v3-monorepo/commit/25d73b3d091f5dde943ad6b7d90db9569222510d

## Deployment preparation tasks checklist:

- [x] Deployments `README` is updated with the new task name and link
- [x] Task `README` has a description and links to the artifacts
- [x] `build-info` is updated with artifacts compiled at the specified commit
- [x] Artifacts are generated from `build-info`
- [x] New task has a working fork test 

## Code checklist:

- [x] The diff is legible and has no extraneous changes 
- [x] Complex code has been commented in tests and scripts
- [x] Fork test block number and test cases are appropriate for this deployment
- [x] The base branch is either `master`, or there's a description of how to merge

## Issue Resolution

N/A
On top of #125; merge first and then rebase.

### CODE REVIEW

File: v3/tasks/20241205-v3-stable-pool/readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# V3 Stable pool
+
+Deployment of the `StablePoolFactory` for Balancer V3.
+Pools from this factory use stable math inspired in Curve stable pools, which is best suited for correlated assets.
+
+## Useful Files
+
+- [Code](https://github.com/balancer/balancer-v3-monorepo/commit/25d73b3d091f5dde943ad6b7d90db9569222510d).
+- [Sepolia testnet addresses](./output/sepolia.json)
```
Comment (joaobrunoah) [inline v3/tasks/20241205-v3-stable-pool/readme.md:9]:
```suggestion
```

This should be added only after deployment

**Review by joaobrunoah — APPROVED** [review]
LGTM! Artifacts are matching the build-info of the monorepo at commit 25d73b3d091f5dde943ad6b7d90db9569222510d

File: v3/tasks/20241205-v3-stable-pool/readme.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# V3 Stable pool
+
+Deployment of the `StablePoolFactory` for Balancer V3.
+Pools from this factory use stable math inspired in Curve stable pools, which is best suited for correlated assets.
```
Comment (EndymionJkb) [inline v3/tasks/20241205-v3-stable-pool/readme.md:4]:
```suggestion
Pools from this factory use stable math, inspired by Curve stable pools, which is best suited for correlated assets.
```

**Comment by joaobrunoah** [comment]
Very minor thing: I believe the title of the PR should be "Deployment preparation: `V3 Stable Pool Factory`", to match the pattern of the weighted pool PR

**Review by EndymionJkb — APPROVED** [review]
I confirm that the code commit is correct, and the build-info matches. Just one readme nit.

**Review by joaobrunoah — APPROVED** [review]

**Review by elshan-eth — APPROVED** [review]

**Review by EndymionJkb — APPROVED** [review]
Guess I'll pile on! Still looks good.

**Pull request merged**